test/apiv2/python/rest_api/test_v2_0_0_network.py

```diff
@@ -169,11 +187,16 @@ def test_crud(self):
         # network prune
         prune_name = f"Network_{random.getrandbits(160):x}"
         prune_create = requests.post(
-            self.podman_url + "/v1.40/networks/create", json={"Name": prune_name}
+            self.podman_url + "/v1.40/networks/create",
+            json={"Name": prune_name},
+            timeout=5
         )
         self.assertEqual(create.status_code, 201, prune_create.text)
 
-        prune = requests.post(self.podman_url + "/v1.40/networks/prune")
+        prune = requests.post(
+            self.podman_url + "/v1.40/networks/prune",
+            timeout=5
```

Comment:
Could we shave the timeouts to 3, or would that be too close for comfort?

Reply:
This commit will go away when I rebase.  The real change is in #13959.  TBH, this timeout isn't really needed, it was added to assist debugging a test-hang.  I'll strip it out.